qemu/tests/cfg/usb.cfg

```diff
@@ -338,6 +338,51 @@
                 - usb_multi_times:
                     no usb_negative_test
                     usb_repeat_times = 20
+        - usb_redir:
+            type = usb_redir
+            only Linux
+            no Host_RHEL.m8.u3
+            usb_devices += " usbredir1"
+            usb_type_usbredir1 = usb-redir
+            chardev_id_usbredir1 = usbredir_chardev1
```

Comment:
Better match with the default value in the vt patch:
```
                chardev_id = usb_dev_params.get("chardev_id",
                                                "%s-chardev" % usb_dev)
```
Then it should be `usbredir1-chardev`.  And refer the comment in the vt patch, it should be `chardev_usbredir1`, can generate it automatically.

Reply:
Updated, thanks!